### PR TITLE
Decreasing stop time distance notice

### PR DESCRIPTION
**Pull request by lilian-hunt**

### CODE REVIEW

**Review by aannie-y — COMMENTED** [review]
lgtm

File: gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/DecreasingStopTimeDistanceNotice.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  public DecreasingStopTimeDistanceNotice(
+      String tripId,
+      long csvRowNumber,
+      int stopSequence,
+      double shapeDistTraveled,
+      long prevCsvRowNumber,
+      int prevStopSequence,
+      double prevShapeDistTraveled) {
+    super(
+        new ImmutableMap.Builder<String, Object>()
```
Comment (KathleenQ) [inline gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/DecreasingStopTimeDistanceNotice.java:31]: You can directly use "ImmutableMap.of(...)" here, to be consistent with all other notices.
No need to have ".put(...)" and ".build()" then.

Comment (lilian-hunt) [reply gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/DecreasingStopTimeDistanceNotice.java:31]: No, not all notices are like this.  "ImmutableMap.of(...)" is limited to 5 entires. See DecreasingShapeDistanceNotice.java that uses the .put and .build

Comment (KathleenQ) [reply gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/DecreasingStopTimeDistanceNotice.java:31]: Sorry that I did not realised that earlier.

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**